src/rest-server/src/models/v2/group.js

```diff
@@ -355,8 +357,8 @@ const deleteNonexistVCs = async () => {
       if (groupItem.extension && groupItem.extension.acls && groupItem.extension.acls.virtualClusters) {
         const checkedVCs = groupItem.extension.acls.virtualClusters.filter((vcname) => vcSet.has(vcname));
         if (checkedVCs.length !== groupItem.extension.acls.virtualClusters.length) {
-          logger.info(`Update group: [${groupItem.groupname}] vc list. 
-          old: ${groupItem.extension.acls.virtualClusters}, new: ${checkedVCs}.`);
+          logger.info(`Update group: ${groupItem.groupname} vc list, ` +
```

Comment:
remove `+`, you can use `\` if you do not want a new line

Reply:
Seems it's not ECMAScript standards?